IFC4x3/Properties/s/SamplingPointType_3snE6alTz82vidqo5JBy_l/Documentation.md

```diff
@@ -0,0 +1 @@
+The sourceLocation is the location from where the MaterialSample was obtained, e.g. outcrop point, tunnel chainage, any other local reference.
```

Comment:
What is "sourceLocation"? Any reference to its definition?

Also, the description seems to not match to the property name.

Reply:
Ok, then I would propose re-writing the definition to something like: "Specifies the location from where the material sample was obtained, e.g. outcrop point, tunnel chainage, any other local reference."

Reply:
This sounds good to me.

Reply:
Done.